src/Process/Builder/MutantProcessBuilder.php

```diff
@@ -49,11 +52,20 @@ class MutantProcessBuilder
 {
     private $testFrameworkAdapter;
     private $timeout;
+    private $eventDispatcher;
+    private $resultFactory;
 
-    public function __construct(TestFrameworkAdapter $testFrameworkAdapter, int $timeout)
-    {
+    // TODO: is it necessary for the timeout to be an int?
```

Comment:
Not at all. Process supports floats.

Reply:
Will probably try to change it in another PR then :)